src/RecipeRepository.js

```diff
@@ -5,12 +5,17 @@ class RecipeRepository {
     this.recipes = data.map(recipe => new Recipe(recipe));
   }
 
-  filterByTag(searchTerm) {
-    return this.recipes.filter(recipe => {
-      if (recipe.tags.includes(searchTerm)) {
-        return recipe.id;
-      }
-    })
+  filterByTag(searchTerms) {
+    const results = [];
+    for (let i = 0; i < searchTerms.length; i++) {
+      this.recipes.filter(recipe => {
+        if (recipe.tags.includes(searchTerms[i])) {
+          results.push(recipe.id);
+        }
+      })
+    }
+
+    return [...new Set(results)];
```

Comment:
Good work getting this filtering to work with multiple tags selected. I don't see right now how to replace the `for` loop but we can put that on the nice-to-have list, as discussed on our call.